### PR TITLE
Create professional-services terms in about.sourcegraph.com

### PR DESCRIPTION
**Pull request by anorrish**

### CODE REVIEW

File: content/terms/premium-support.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Last Modified: May 5, 2023
+
+This document outlines the basic terms and conditions that apply to Sourcegraph's Premium Support Offerings.
+
+## Premium Support
+
+### Benefits of Premium Support
+Purchasing Premium Support provides the following services beyond those provided by Sourcegraph's standard support model:
+- **Accelerated Service Level Agreements (SLAs)** - Accelerated triaging, faster initial response times, and queue priority lead reduced times to resolution.
+- **Elevated Support Expertise** - Named, dedicated primary and backup senior support engineers possessing deep knowledge and understanding of you specific organizations setup and tech stack with the ability to assist in the resolution of issues that arise.
```
Comment (enriquegh) [inline content/terms/premium-support.md:15]: We are no longer referring it as primary and backup but simply two senior support engineers.

File: content/terms/premium-support.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Last Modified: May 5, 2023
+
+This document outlines the basic terms and conditions that apply to Sourcegraph's Premium Support Offerings.
+
+## Premium Support
+
+### Benefits of Premium Support
+Purchasing Premium Support provides the following services beyond those provided by Sourcegraph's standard support model:
+- **Accelerated Service Level Agreements (SLAs)** - Accelerated triaging, faster initial response times, and queue priority lead reduced times to resolution.
```
Comment (enriquegh) [inline content/terms/premium-support.md:14]: This is true except the faster initial response times. The times we hold ourselves to will remain the same but we will bump the tickets to top of the queue which will mean faster time for someone to pick it up.

File: content/terms/implementation-services.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+---
+
+Last Modified: May 5, 2023
+
+This document outlines the basic terms and conditions that apply to Sourcegraph's Implementation Services Offerings.
+
+## Implementation Services Terms of Service
+
+### Implementation Services Description
+Dedicated, paid Implementation Services to design, plan, and provide hands-on assistance for Sourcegraph self-hosted production deployments projects. This service will:
```
Comment (amenne) [inline content/terms/implementation-services.md:13]: Hands-on (via direct access) or over the shoulder?

File: content/terms/premium-support.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+### Premium Support SLAs
+
+1st response times for both Standard and Premium Support remain the same according to our [SLAs](https://handbook.sourcegraph.com/departments/technical-success/support/#slas). However, Emergency and Severe Impact issues are supported 24 x 7 with Premium Support (versus 24 x 5 with Standard Support).
+
+### Premium Support Packages
+
+Each Premium Support Package is priced independently. For pricing, please discuss with your Sourcegraph contacts. The current Packages offered are:
+- Package 1: Premium SLAs
+- Package 2: Dedicated Support (note - purchasing <em>Package 2: Dedicated Support</em> comes with <em>Package 3: Slack Support</em>)
+- Package 3: Slack Support
```
Comment (amenne) [inline content/terms/premium-support.md:28]: Remind me, does this mean _only_ support via slack? What about Sales/TA wanting to have a slack channel for comms? Where did we land there?

File: content/terms/implementation-services.md (hunk 3 above)
Comment (anorrish) [reply content/terms/implementation-services.md:13]: Over the shoulder. Good Catch.

File: content/terms/premium-support.md (hunk 4 above)
Comment (enriquegh) [reply content/terms/premium-support.md:28]: We leaned towards TA as well but never got confirmation whether not having Premium Support package 2 meant a TA would not be there at all. cc @billCaplan

**Pull request merged**